iac-extensions/spring-config/src/main/antlr/PropertiesParser.g4

```diff
@@ -27,7 +46,7 @@ eol
     ;
 
 commentText
```

Comment:
There can be another vertical order: `comment`, `commentStartAndText` and `commentText`